doc/ensemble.rst

```diff
@@ -100,13 +100,29 @@ takes the same parameters than the scikit-learn
   ...                                 ratio='auto',
   ...                                 replacement=False,
   ...                                 random_state=0)
-  >>> bbc.fit(X, y) # doctest: +ELLIPSIS
+  >>> bbc.fit(X_train, y_train) # doctest: +ELLIPSIS
   BalancedBaggingClassifier(...)
   >>> y_pred = bbc.predict(X_test)
   >>> confusion_matrix(y_test, y_pred)
-  array([[  12,    0,    0],
-         [   1,   54,    4],
-         [  49,   53, 1077]])
+  array([[   9,    1,    2],
+         [   0,   55,    4],
+         [  42,   46, 1091]])
+
+It also possible to turn a balanced bagging classifier into a balanced random
+forest using a decision tree classifier and setting the parameter
+``max_features='auto'`. It allows to randomly select a subset of features for
```

Comment:
missing backtick after auto

Reply:
Thanks Andy

Reply:
Thanks for adding the example, I think it's very helpful! (I just linked my students to it ;)

Reply:
We will try to find sometime to check the sickit-learn PR to push it further